api/endpoints.go

```diff
@@ -11,6 +11,8 @@ var Endpoints = []rest.Endpoint{
 	nodesCmd,
 	nodeCmd,
 	terraformStateCmd,
-        terraformLockCmd,
-        terraformUnlockCmd,
+	terraformLockCmd,
+	terraformUnlockCmd,
+	jujuusersCmd,
+	jujuuserCmd,
```

Comment:
Nit but could we use

jujuUsersCmd
jujuUserCmd

for consistency with other endpoint names

Reply:
actually ignore me - I see this is consistent with the path being used.